app/utils.py

```diff
@@ -51,3 +80,27 @@ def get_coordinates(city: str) -> dict:
 
         time.sleep(current_delay)
         current_delay *= 2
+
+
+def validate_image(stream):
```

Comment:
Request validation is always a good idea and checking that you have the right type of file is great first step. 

Additionally you could check for a maximum size, the right aspect ratio (in order to not break the frontend layout). 

It really depends on who you're going to expose a file upload to. If it's the average user it's always good to assume the worst 😂